crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -69,8 +74,47 @@ fn nonce_storage_key<T: AddressBytes>(address: T) -> String {
 pub(crate) trait StateReadExt: StateRead + crate::assets::StateReadExt {
     #[instrument(skip_all)]
     async fn get_account_balances(&self, address: Address) -> Result<Vec<AssetBalance>> {
```

Comment:
I wish this simpler named function was done in terms of ibc prefixes because that's the "natural" representation inside sequencer (as it maps all trace prefixed items to ibc prefixed and only makes the reverse mapping at the API boundary).

Reply:
changed `get_account_balances()` to `get_account_balances_traced_prefixed()` and used the simpler name for `get_account_balances_ibc_prefixed()` 👍